Update vnet-deployment.md (#1353)
MicrosoftDocs/dataexplorer-docs: data-explorer/vnet-deployment.md

### Configure Network Security Group rules using subnet delegation

> Configuration of Network Security Group rules using [subnet delegation](/azure/virtual-network/subnet-delegation-overview) is the default for Virtual Network injected Azure Data Explorer clusters. To do so, you must delegate the subnet to *Microsoft.Kusto/clusters* before creating the cluster in the subnet.

Reviewer: ```suggestion
[Subnet delegation](/azure/virtual-network/subnet-delegation-overview) is the default method for configuring Network Security Group rules for Azure Data Explorer clusters deployed into a subnet in your virtual network. When using subnet delegation, you must delegate the subnet to *Microsoft.Kusto/clusters* before creating the cluster in the subnet.
```

By enabling subnet delegation on the cluster's subnet, you enable the service to define its pre-conditions for deployment in the form of Network Intent Policies. When creating the cluster in the subnet, the NSG configurations mentioned in the following sections are automatically created for you.

 > [!WARNING]

>  > Your Azure Data Explorer cluster will stop working eventually if you change the subnet delegation configuration. This might mean that it will not be able to start after it was stopped or we will not be able to apply management or health monitoring on the cluster.

Reviewer: ```suggestion
  > Changing your subnet delegation configuration will eventually disrupt the normal operation of your cluster. For example, after stopping the cluster you may not be able to start your cluster, run management commands, or apply health monitoring on your cluster.
```

Reviewer: How do you fix remedy this situation?
